include/z64environment.h

```diff
@@ -110,28 +184,27 @@ typedef struct {
     /* 0xB0 */ f32 windSpeed;
     /* 0xB4 */ u8 numLightSettings;
     /* 0xB8 */ EnvLightSettings* lightSettingsList; // list of light settings from the scene file
-    /* 0xBC */ u8 blendIndoorLights; // when true, blend between indoor light settings when switching
-    /* 0xBD */ u8 unk_BD; // indoor light index
-    /* 0xBE */ u8 unk_BE; // prev indoor light index?
-    /* 0xBF */ u8 unk_BF;
-    /* 0xC0 */ EnvLightSettings lightSettings;
-    /* 0xD6 */ u16 unk_D6;
-    /* 0xD8 */ f32 unk_D8; // indoor light blend weight?
-    /* 0xDC */ u8 unk_DC;
-    /* 0xDD */ u8 gloomySkyMode;
-    /* 0xDE */ u8 unk_DE; // gloomy sky state
-    /* 0xDF */ u8 lightningMode;
-    /* 0xE0 */ u8 unk_E0; // env sounds state
+    /* 0xBC */ u8 lightBlendEnabled; // only used with `LIGHT_MODE_SETTINGS` or on override
+    /* 0xBD */ u8 lightSetting; // only used with `LIGHT_MODE_SETTINGS` or on override
+    /* 0xBE */ u8 prevLightSetting;
+    /* 0xBF */ u8 lightSettingOverride;
+    /* 0xC0 */ EnvLightSettings lightSettings; // settings for the currently "live" lights
+    /* 0xD6 */ u16 lightBlendRateOverride;
+    /* 0xD8 */ f32 lightBlend;
```

Comment:
if I understand how you have placed the "only used with" comments it's on the main variables?
```suggestion
    /* 0xD8 */ f32 lightBlend; // only used with `LIGHT_MODE_SETTINGS` or on setting override
```

Reply:
that was my intention yeah, it got kind of spammy putting it on every one. Hoping its obvious which variables go with each other, tho it might be tough.
But yea ill make this change